randovania/data/json_data/prime1.txt

```diff
@@ -2366,6 +2570,26 @@ Asset id: 2263559682
 > Dock to Life Grove Tunnel; Heals? False
   > Life Grove Tunnel/Dock to Life Grove
       Trivial
+  > Life Grove/Pickup (X-Ray Visor)
+      Trivial
+
+> Pickup (X-Ray Visor); Heals? False
+  > Life Grove/Behind PB wall
+      X-Ray Visor and Can lay power bomb
```

Comment:
I don't think X-Ray is necessary for this honestly, even though you're supposed to use to see beyond. Not really a trick, but that's up for debate.

Reply:
Sounds like the perfect example of "Knowledge (Beginner)"